### ISSUE TITLE
How to re-login / re-install GitLab using Visual Studio?

### ISSUE THREAD

**Issue by senad004** [maikebing/GitLab.VisualStudio#60]
How to re-login / re-install GitLab using Visual Studio?

**Comment by cbeaulieu-gt** [comment]
The extension stores the login information in the Windows Credential Manager. If you go to it and then choose **Windows Credentials** then scroll down and find **Generic Credentials**. Under that list scroll down and look for a GitLab domain. There should be a few entries for gitlabs.com. Just go ahead and delete those entries (note that if another application uses the same authentication method you may end up deleting those as well but as far as I can see there is not way to associate the credential with the creating application). 

Once you go back to Visual Studio and try to use the extension you should be prompted for login when you try to push/pull or do other server related activities.

**Comment by Travisivart** [comment]
I was able to login to my local gitlab initially using the api key.

However, when I attempted to create a repo, there was a popup for my username and password,
I entered this but had a typo in the password/api key. This resulted in an error and it continued to fail to connect. I followed the above steps to remove the Generic Credentials, which I was able to remove. However, when I go to visual studio now, the GitLab option no longer appears under connect. I tried uninstalling the extention and reinstalling it and it still will not appear.
